recipes-extended/libtirpc/libtirpc_%.bbappend

```diff
@@ -0,0 +1 @@
+CFLAGS += "-fPIC"
```

Comment:
Why is this needed?

Reply:
pyicbinn links against some RPC related .a files, and if they aren't compiled with -fPIC pyicbinn will fail to build.
BUT the .a files in question are compiled with -fPIC. Turns out that if you link against other .a files that aren't compiled with -fPIC the flag is (in effect) ignored.
The RPC files in question are linked against libtirpc, which doesn't normally compile with -fPIC. All resulting object files from libtirpc were found to require -fPIC in order for it to propagate up to pyicbinn.

Reply:
@Tmcg2 thanks for the detailed information.  That would be great to include in the commit message and maybe also a comment in the bbappend.

Reply:
Added the extra info in the commit message